web/packages/core/src/ruffle-player.ts

```diff
@@ -780,42 +811,125 @@ export class RufflePlayer extends HTMLElement {
         let issueLink = `https://github.com/ruffle-rs/ruffle/issues/new?title=${encodeURIComponent(
             issueTitle
         )}&body=`;
-        let issueParameters = encodeURIComponent(errorText);
+        let issueBody = encodeURIComponent(errorText);
         if (
             errorArray.stackIndex > -1 &&
-            String(issueLink + issueParameters).length > 8195
+            String(issueLink + issueBody).length > 8195
         ) {
             // Strip the stack error from the array when the produced URL is way too long.
             // This should prevent "414 Request-URI Too Large" errors on Github.
             errorArray[errorArray.stackIndex] = null;
-            issueParameters = encodeURIComponent(errorArray.join(""));
+            issueBody = encodeURIComponent(errorArray.join(""));
         }
-        issueLink += issueParameters;
+        issueLink += issueBody;
 
         // Clears out any existing content (ie play button or canvas) and replaces it with the error screen
+        let errorBody, errorFooter;
+        switch (errorIndex) {
+            case 100:
```

Comment:
Could we use a TypeScript enum to represent these error codes:
https://www.typescriptlang.org/docs/handbook/enums.html